dbfit-java/core/src/main/java/dbfit/api/DbCommand.java

```diff
@@ -0,0 +1,25 @@
+package dbfit.api;
+
+import java.sql.SQLException;
+
+/**
+ * An object representing an executable database operation
+ * with no result returned.
```

Comment:
Should we loosen our definition here and re-address it when we implement for of PR #463?